03-events-forms-refs-life-cycle/forms-and-refs/add_divs/src/DivForm.js

```diff
@@ -0,0 +1,76 @@
+import React, {Component} from 'react';
+
+//one form is one div!!
+class DivForm extends Component{
+    constructor(props){
+    	super(props);
+    	
+    	this.state = {
+            height : '',
+            width :'',
+            backgroundColor : ''
+    	}
+    	this.handleChange = this.handleChange.bind(this);
+    	this.handleSubmit = this.handleSubmit.bind(this);
+    }//end constructor
+
+    handleChange(e){
+    	// debugger
+        this.setState({[e.target.name] : e.target.value});
+    }
+    //one form is one div!!
+    handleSubmit(e){
+    	// debugger
+        e.preventDefault();
+        this.props.addDiv(this.state);
+        //this.width.focus();
+    }
+
+    render(){
+    	return(
+            <div id="con">
+                <form onSubmit = {this.handleSubmit}>
+                     <header>
+                        <h1>Creating Div's </h1>
+                        <p>Please enter the details:</p>
+                     </header>
+                    <div id="inputFields">
+	                    <div>    
+		                    <input
+		                        type = "number"
```

Comment:
just a tiny nitpick, but usually there are no spaces around `=` in JSX. Try to use prettier if you can.